test/fixtures/box/noDefaultScales.js

```diff
@@ -3,7 +3,8 @@ module.exports = {
     type: 'bar',
     data: {
       datasets: [{
-        data: [0, 5, 10, 15, 20, 22]
+        data: [0, 5, 10, 15, 20, 22],
+        yAxisID: 'y1'
```

Comment:
Maybe these test are an unsupported use case with v4? If so, maybe just remove the tests.

Reply:
Not sure because I think it's still valid. Let me do a test and then I'll come back to you. My doubt it' related to index axis.

Reply:
Done some tests and these tests could be removed. But with these tests we can tests that the `retrieveScaleID` function is working well with no defaults scale ids as well. Let me know!

Reply:
I think I can try to move the tests to a unit tests (no fixture) even if I have to think a little about what to test.

Reply:
For me removing the tests from this pr and adding new tests later for `retrieveScaleID` is ok.

Reply:
I have already done. Removed noDeault fixture, added retrieveScaleID test cases in helpers spec.

Reply:
Have a look if looks good

Reply:
It's better to acquire a chart and use the scales of the chart instead of a custom scales structure